jsx/Form.js

```diff
@@ -1334,13 +1353,20 @@ class StaticElement extends Component {
     super(props);
   }
   render() {
-    return (
-      <div className="row form-group">
+    let label = null;
+    if (this.props.label) {
+      label = (
         <label className="col-sm-3 control-label">
           {this.props.label}
         </label>
-        <div className="col-sm-9">
-          <p className="form-control-static">{this.props.text}</p>
+      );
+    }
+
+    return (
+      <div className="row form-group">
+        {label}
+        <div className={this.props.class}>
+          <p className={this.props.textClass}>{this.props.text}</p>
```

Comment:
@zaliqarosli if you take a look at login or checkout my PR, you will notice why I added the changes to the jsx/Form.js. As an example the styles & classes for the input fields in the form needed to be modified to display the login fields as perfectly as before. Otherwise the labels being removed (or even kept) would look ugly.